internal/operators/cnv/manifest.go

```diff
@@ -57,6 +58,14 @@ func Manifests(config Config) (map[string][]byte, []byte, error) {
 
 	openshiftManifests := make(map[string][]byte)
 
+	if isSingleNodeCluster && config.SNOInstallHPP {
+		cnvHpp, err := hpp(config.SNOPoolSizeRequestHPPGib)
+		if err != nil {
+			return nil, nil, err
+		}
+		openshiftManifests["99_openshift-cnv_hpp.yaml"] = cnvHpp
```

Comment:
Suggesting to avoid using 99 because it will prevent form the user the option to override this config.

Reply:
We use this prefix for all the operator manifests. Do you want us to change it everywhere (work for other PR)?

Reply:
We use this throughout quite often in other operators (OCS/LSO) as well:
https://github.com/openshift/assisted-service/blob/master/internal/operators/ocs/manifest.go#L53

Reply:
It can be done in a different PR, but it's something that needs to be handled, 99 will mean that any configuration that user will try to override will be ignored.